jaxws-ri/rt/src/main/java/com/sun/xml/ws/util/xml/XmlUtil.java

```diff
@@ -87,6 +87,8 @@ public Boolean run() {
             }
     );
 
+    private static boolean RUN_WITH_SECURITY_MANAGER = (System.getSecurityManager() != null);
```

Comment:
there may be a chance that SM will be changes during runtime - do we have to care about such case?